tests/e2e/framework/kubernetes.go

```diff
@@ -562,13 +568,30 @@ func (k *KubeInfo) Teardown() error {
 			} else {
 				istioYaml = getClusterWideInstallFile()
 			}
-
+		}
+		if *useOperator {
+			// Need an operator unique delete procedure
+			if _, err := util.Shell("kubectl -n %s get IstioControlPlane example-istiocontrolplane -o=json | jq '.metadata.finalizers = null' | kubectl -n %s apply --kubeconfig=%s -f -",
```

Comment:
this doesn't look right.. this isn't how we tell users to remove the operator is it?

Reply:
it is documented as such in the doc pr here: https://deploy-preview-5146--preliminary-istio.netlify.com/docs/setup/install/standalone-operator/

Reply:
actually this way to patch finalizer doesn't work for me. Is the path correct?

Reply:
this has been resolved by simply deleting the cr associated with the installation.